contents/blog/sunsetting-helm-support-posthog.md

```diff
@@ -0,0 +1,59 @@
+---
+date: 2023-01-05
+title: "Sunsetting Kubernetes support for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["tim-glaser"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
```

Comment:
@joethreepwood what should we do here?

Reply:
I'll speak with @lottiecoxon about if we can create some dedicated art by Wednesday

Reply:
(Lottie is going to save the day and create a new card for this)

Reply:
<img width="1200" alt="posthog-bye-kubernetes" src="https://user-images.githubusercontent.com/84011561/217006029-a079bcd9-8308-4cf1-9c02-1146fc5c3463.png">

 Committing in